.env.sh

```diff
@@ -34,8 +34,8 @@ export RAFIKI_IMAGE_WORKER=rafikiai/rafiki_worker
 export RAFIKI_IMAGE_PREDICTOR=rafikiai/rafiki_predictor
 
 # Docker images for dependent services
-export IMAGE_POSTGRES=postgres:10.5
-export IMAGE_REDIS=redis:5.0-rc
+export IMAGE_POSTGRES=postgres:10.6-alpine
```

Comment:
do you need any features from 10.6-alpine?
is it a stable version?

Reply:
No new features required. Will set it to `10.5-alpine` then.

With regards to why i change these images to their `alpine` variants: https://nickjanetakis.com/blog/the-3-biggest-wins-when-using-alpine-as-a-base-docker-image. Their services should not be affected at all.

Reply:
10.6 is fine.
my concern was about alpine.